UnicodeJsps/src/main/java/org/unicode/jsp/UnicodeJsp.java

```diff
@@ -208,28 +208,39 @@ public static void getDifferences(
         UnicodeUtilities.getDifferences(setA, setB, abbreviate, abResults, abSizes, abLinks);
     }
 
-    public static int parseCode(String text, String nextButton, String previousButton) {
+    public static int[] parseCode(String text, String nextButton, String previousButton) {
```

Comment:
observation: I'm unopinionated on the return type, and what you have seems reasonable. I don't know if matters whether it is an int array `int[]` or a collection type object that implements `List<T>`.